maven-core/src/main/java/org/apache/maven/internal/aether/DefaultRepositorySystemSessionFactory.java

```diff
@@ -197,6 +197,15 @@ else if ( request.isUpdateSnapshots() )
         for ( Server server : decrypted.getServers() )
         {
             AuthenticationBuilder authBuilder = new AuthenticationBuilder();
+            //MNG-5583 per endpoint PKI authentication
+            authBuilder.addString( "getKeyAlias", server.getKeyAlias() );
```

Comment:
Can't you re-use the constants you declared in one of the earlier pull requests here?